style.css

```diff
@@ -250,3 +650,349 @@ body {
+  gap: 30px;
+}
+
+.card-grid {
+  display: grid;
+  grid-template-areas:
+    "card-img card-img card-img"
+    "card-title card-title card-title"
+    "tags tags tags"
+    "button button button";
+  grid-template-rows: 8px 1fr 7px;
+  grid-template-columns: 20% 1fr 15%;
+  grid-row-gap: 5px;
+  grid-column-gap: 5px;
+  height: 30vh;
+  margin: 0;
+}
+
+/* Card styling */
+.card {
+  position: relative;
+  overflow: hidden;
+  border-radius: 10px;
+  padding: 4px 2px;
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  border: 1px solid #ccc;
+}
+
+.card .card-img {
+  grid-column: 1 / 4;
+  background-image: url('./assets/images/img-placeholder.png');
+  background-size: cover;
+  background-position: center;
+  height: 200px;
+  object-fit: cover;
+}
+
+.card-content {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px;
+  background-color: #fff;
+  transition: transform 0.3s ease-out;
+  align-items: center;
+  text-align: center;
+}
+
+.card-title {
+  color: #3a4a42;
+  width: 100%;
+  height: 57px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card .overlay h3 {
+  word-wrap: break-word;
+}
+
+/* .tags{ */
+
+.tags {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-top: 10px;
+  border-radius: 4px;
+  margin-bottom: 30px;
+  margin-left: 0;
+}
+
+.tags span.works {
+  background-color: #c6c6c6;
+  border-radius: 6px;
+  padding: 4px 8px;
+  margin-right: 5px;
+  color: black;
+}
+
+.tags span.about {
+  background-color: #fff;
+  border-radius: 6px;
+  padding: 4px 8px;
+  margin-right: 5px;
+  color: #36b37e;
+}
+
+.tags span:first-child {
+  margin-left: 0;
+}
+
+.tags span:last-child {
+  margin-right: 0;
+}
+
+.project-btn {
+  position: relative;
+  background-color: #36b37f;
+  color: #fff;
+  padding: 8px 18px;
+  border-radius: 5px;
+  border: none;
+  cursor: pointer;
+  font-size: 11px;
+  bottom: 8px;
+  width: 122px;
+  height: 48px;
+  align-items: center;
+  display: block;
+  margin: 0 auto;
+}
+
+/* about me  */
+
+.about-me .project-btn::after {
+  content: "";
+  position: absolute;
+  top: -10px;
+  left: -150px;
+  width: 50px;
+  height: 50px;
+  background-image: url('./assets/images/about-bg-left.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.header-content h1:nth-of-type(2) {
+  color: #70c572;
+}
+
+/* about me */
+.aboutme-container {
+  position: relative;
+  width: 100%;
+  max-width: 600px;
+  text-align: center;
+  z-index: 1;
+  align-items: center;
+  margin: auto;
+  padding: 0 18px;
+  margin-bottom: 50px;
+}
+
+.card-img-content {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px;
+  background-color: #fff;
+  transition: transform 0.3s ease-out;
+  align-items: center;
+  text-align: center;
+}
+
+/* About card */
+.about-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please stick to the Figma design but still there are some spacing issues so kindly fix it :+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

| Your Design | Figma Design |
|--------|--------|
| ![Screenshot from 2023-04-30 04-34-59](https://user-images.githubusercontent.com/45226848/235328356-f8eada7d-bd5e-45d1-9c44-b1fd6d8cf8af.png) | ![Screenshot from 2023-04-30 04-37-18](https://user-images.githubusercontent.com/45226848/235328361-8278efcc-9ecb-400c-92db-d668b1e52eca.png) |  

- [ ] Please add the same data as mentioned in Figma design like in `languages`: `JavaScript`, `Ruby`, `HTML` and `CSS` and in `Frameworks`: `React`, `Ruby On Rails`, `RSpec`, etc. and also for `Skill` add same data as mentioned in Figma design :+1: